base/errorshow.jl

```diff
@@ -110,6 +110,13 @@ function show_error_hints(io, ex, args...)
     end
 end
 
+show_index(io::IO, x::Any) = show(io, x)
+show_index(io::IO, x::Slice) = show_index(io, x.indices)
+show_index(io::IO, x::LogicalIndex) = show_index(io, x.mask)
```

Comment:
how would that handle the case when `x.mask` is very long? Do we show then it compactly? (I guess we do but I just want to be sure)

Reply:
Yes I get this:
```julia
julia> x[10, fill(true, 1000)]
ERROR: BoundsError: attempt to access 2×2 Array{Float64,2} at index [10, Bool[1, 1, 1, 1, 1, 1, 1, 1, 1, 1  …  1, 1, 1, 1, 1, 1, 1, 1, 1, 1]]
```